Python/Product/Analysis/scrape_module.py

```diff
@@ -412,7 +422,21 @@ def _init_restype_fromdocstring(self):
             return "return dict()"
         if typeName.startswith('('):
             return "return tuple()"
-        if 'Return a string' in first_line:
+        if typeName.startswith('date'):
+            return "return datetime(1, 1, 1)"
+        if typeName.startswith('time'):
+            return "return datetime(1, 1)"
+        if typeName.startswith('unicode'):
+            return "return unicode('')"
+        if typeName.startswith('index'):
+            return "return 1"
+        if typeName.startswith('bool'):
+            return "return True"
+        if typeName.startswith('true') or typeName.startswith('True'):
```

Comment:
Python supports passing a tuple to `startswith`, so this could be written as: `typeName.startswith(('true', 'True'))`

But see my above comment - I don't think this is the right approach to take here.

Reply:
Yeah, I can just leave original code (i.e. handling `->` only). However, then `datetime.now().` does not bring any completions in V2